tasks/main.yml

```diff
@@ -1,28 +1,34 @@
 ---
-- name: Add Nginx Rsyslog configuration file
-  template: src="{{ item }}" dest="/etc/rsyslog.d/{{ item }}" owner=root group=root
-  with_items:
-    - 21-nginx-loggly.conf
-    - 21-rails.conf
-    - 22-loggly.conf
+- fail: msg="This role requires 'loggly.token'"
+  when: loggly.token | trim == ''
 
-- name: Create cerfiticates folder If It Does Not Exist
-  file: path=/etc/rsyslog.d/keys/ca.d state=directory
+# Only supported by Ubuntu targets - will provide support for $InputFileName wilcard paths
```

Comment:
i'll change it to:

          - name: "Install rsylog 8 for $InputFileName wilcard paths"
            apt_repository: repo="{{ loggly.rsylog_ppa }}"
            when: ansible_distribution == "Ubuntu" and loggly.rsyslog_version|int > 7